README.md

```diff
@@ -1,5 +1,10 @@
 ![SpatialOS Unreal SDK](docs/assets/unreal-sdk-header.png)
 
+**New from November 2018: The SpatialOS GDK for Unreal**<br/>
```

Comment:
This is a brilliant idea. Maybe we should even make it more obvious? Larger header perhaps